js/packages/components/items/floating/FloatingToggleMenuCustomPack.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+
+import { ToggleItemMenuWithIconProps } from '../interfaces'
+import { FloatingToggleMenuPriv } from './FloatingToggleMenu.priv'
+
+export const FloatingToggleMenuCustomPack: React.FC<ToggleItemMenuWithIconProps> = props => {
```

Comment:
I would prefer to pass PackProps instead of having the *CustomPack variants

Reply:
I agree too for this one, because with the time we'll have only one icon pack, and it's more easily to delete the props than a file, an export, and replace the component in screen 👍